apps/gadgetron/CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 configure_file(gadgetron_config.in gadgetron_config.h)
 
+include_directories(${PUGIXML_INCLUDE_DIRS})
```

Comment:
This is not needed and will break CMakes automatic include dependency management.

Reply:
In windows, without this line, it will have including error.